docs/Usage/README.md

```diff
@@ -0,0 +1,6 @@
+# API Usage
+
+The following documents provide sample code snippets for using the public APIs provided by the components in aepsdk-core-android.
+
+- [MobileCore](./MobileCore.md)
+- [Identity](./Identity.md)
```

Comment:
Can you also add 2 more pages for Signal and Lifecycle which talks about Registration and extensionVersion APIs

Reply:
Can you also link Lifecycle & Signal pages from this page?

Reply:
Whops! Added them now. 😅